resolwe_bio/processes/support_processors/mutations_table.py

```diff
@@ -240,6 +241,40 @@ def get_depth(variants_table, bam):
     return variants_table.to_csv("mutations.tsv", sep="\t")
 
 
+def encode_variant_genotype(row):
+    """Encode variant to genotype notation.
+
+    Variants are given as <allele1>/<allele2>, e.g. T/T or C/T
+
+    Encode these variants as:
+        - 0/1 for heterozygous variants
+        - 1/1 for homozygous variants
+    """
+    try:
+        allele_line = row.get("genotype", np.nan)
+        print("allele_line", allele_line)
+        allele_re = r"^([ACGT]+)/([ACGT]+)$"
+        allele1, allele2 = re.match(allele_re, allele_line).group(1, 2)
+    except AttributeError:
+        # AttributeError is raised when there is no match, e.g.
+        # there is a string value for column "genotype" but
+        # the above regex can't parse it
+        print(f'Cannot encode variant from value "{allele_line}".')
+        return np.nan
+
+    if allele1 == allele2 == row["alternative"]:
+        return "1/1"
+    else:
+        return "0/1"
+
+
+def extract_alt_allele_depth(row):
+    """Extract the depth of the alternative allele from the AD field."""
+    ad_line = row.get("unfiltered_allele_depth")
```

Comment:
When key must be present, I suggest you use `row["..."]` instead of the `get` method.